to_do_app/lib/home/myHomePage.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:to_do_app/home/listItem.dart';
+
+import '../app_router.dart';
+import '../domain/card_to_do.dart';
+import '../repository/repository_task.dart';
+
+class MyHomePage extends StatefulWidget {
+  const MyHomePage({super.key, this.title = 'Test'});
+
+  final String title;
+
+  @override
+  State<MyHomePage> createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  final repository = RepositoryTask();
+  late var _elements;
+  final _saved = <CardToDo>{};
+
+  @override
+  void initState() {
+    // TODO: implement initState
+    _elements = repository.getTasks();
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Row(
+          children: [
+            Expanded(child: Text(widget.title, textAlign: TextAlign.center)),
+            IconButton(onPressed: _addElement, icon: const Icon(Icons.add))
+          ],
+        ),
+      ),
+      body: ListView.separated(
+        padding: const EdgeInsets.all(16.0),
+        itemCount: _elements.length,
+        itemBuilder: (context, index) {
+
+          final alreadySaved = _saved.contains(_elements[index]);
```

Comment:
i think that you are not using this

Reply:
The attribute 'alreadySaved' I already deleted, but the itemCount and itemBuilder if I delete them the list view throw error